juju/juju
# Relation init leader

## Description of change

When there is more than one unit, only the leader should attempt to set the relation status when the unit agent initialises or a new relation is created.

## QA steps

bootstrap, add mysql, add extra unit
create relation
check for errors and that relation status is set

## Bug reference

https://pad.lv/1719328
